api-server/src/authentication/parse-jwt.ts

```diff
@@ -1,10 +1,6 @@
-import * as fs from "fs";
 import jwt = require('jsonwebtoken');
 import {Request} from "express";
-import * as path from 'path';
-
-const PATH_TO_PEM = process.env.JWT_PEM || './keys/1nfinite.pem';
-const SECRET = fs.readFileSync(path.resolve(PATH_TO_PEM));
+import {SECRET} from "../constants";
```

Comment:
So we don't need the PEM at all anymore?

Reply:
we still need the it's just being imported and potentially defaulted inside of constants now so that we don't have to repeat the defaulting logic